pkg/cosign/cosign.go

```diff
@@ -347,29 +347,29 @@ func matchSubjectAndIssuer(signatures []oci.Signature, subject, issuer string) e
 	if subject == "" && issuer == "" {
 		return nil
 	}
-
+	var s string
```

Comment:
@Namanl2001 - any reason to move this here? The intent was that the issuer and subject are in the same cert and not across two different certs.

Reply:
I did this to print this [message](https://github.com/kyverno/kyverno/pull/3112/files#diff-19f451f34e955fe33fac7e12f3099cc5eb3b05252fc467109736696b3e971265R372) which is outside the loop. also for each cert, we are defining the subject and issuer [here](https://github.com/kyverno/kyverno/pull/3112/files#diff-19f451f34e955fe33fac7e12f3099cc5eb3b05252fc467109736696b3e971265R361-R362) separately so would it be a problem? 

as an alternate, we can declare a new var to use it for the error message purpose

Reply:
I see, it should be fine!